src/test/java/TripListedPage.java

```diff
@@ -1,61 +1,68 @@
+    //take the trip name from each trip on listing page
+
     @Test()
     public static void VerifyCountOfDay() throws InterruptedException {
         WebDriver driver = new ChromeDriver();
         driver.manage().window().maximize();
         driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS) ;
 
-        driver.get("https://www.viator.com/");
-        WebElement textBox = driver.findElement(By.cssSelector("[data-automation='typeahead-input']"));
-        textBox.sendKeys("Los Angeles");
-        textBox.sendKeys(Keys.ENTER);
-//        driver.findElement(By.cssSelector("[data-automation='typeahead-button']")).click();
+        driver.get("https://deens.com/");
+        driver.findElement(By.cssSelector("a[href*='fun-getaway-with-friends-in-london']")).click();
 
-        List<WebElement> tripTitles = driver.findElements(By.cssSelector("[data-shelf-type='TTD_RECOMMENDED_FOR_YOU_SHELF']"));
-        Assert.assertTrue(tripTitles.size() > 0);
+        Thread.sleep(5000);
 
-        //List<WebElement> tripTitles = driver.findElements(By.cssSelector("[class *='Itinerary__ServiceTitle']"));
+        driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS) ;
 
+        List<WebElement> tripTitles = driver.findElements(By.cssSelector("[class*='Itinerary__ServiceTitle']"));
 //        for (int i = 0; i < tripTitles.size(); i++) {
-//            WebElement day = tripTitles.get(i);
-//
-//            String dayText = day.getText();
-//            System.out.println(dayText);
+//            WebElement title = tripTitles.get(i);
+//            String titleText = title.getText();
+//            System.out.println(titleText);
 //        }
-//
-//        Assert.assertTrue(tripTitles.size() > 0);
-//
-//        driver.findElement(By.cssSelector("a[href*='href=\'/book/activity/central-london-pub-crawl-with-skip-the-line-entry-and-guide-in-london_5bdba19ef1342e59cb0f928a\"']")).click();
-//
-//        driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
-////
-//        WebElement pdpTitle = driver.findElement(By.cssSelector("[class*='Service_HeaderWrap']"));
-//        System.out.println(pdpTitle);
-//
-//        Assert.assertEquals(pdpTitle, "SoHostel");
+        Assert.assertTrue(tripTitles.size() > 0);
+
+        System.out.println("tripTitles.get(0) " + tripTitles.get(0).getText());
+        System.out.println("tripTitles.get(0) " + tripTitles.get(0));
+
+
+
+        //then click to individual trip name link you will be landed on the Trip details page, like:
+        //  https://deens.com/book/accommodation/sohostel-in-london_5bb61189a965f27332f9ea7a
+        for (int i = 0; tripTitles.size() > i; i++) {
+            tripTitles = driver.findElements(By.cssSelector("[class*='Itinerary__ServiceTitle-fHxCEn kubSIN'] a"));
+            System.out.println("Number is " + i);
+
+            driver.manage().timeouts().implicitlyWait(20, TimeUnit.SECONDS) ;
```

Comment:
you have already setted up your an 'implicitlyWait' earlier. What is the reason to set up it second time?